test/datahike/test/query_stats_test.cljc

```diff
@@ -51,14 +41,10 @@
           :rules  {}
           :stats  [{:clause '[?e :age ?a]
                     :rels   [{:bound #{'?a '?e} :rows  6}]
-                    :t      :measurement
-                    :type   :lookup
```

Comment:
It is fine for me to remove this information, in particular if it is constant. I think @jsmassa added it for Datomic compatibility though, but I am not sure.

Reply:
Oh, you are right: I believe the `:type :lookup` key-value-pair should still be there, but the the `:lookup-stats :valid` can be removed. I updated `query.cljc` so that we keep `:type :lookup`. This is the behaviour that we used to have in the past, which can be seen by running

```
git diff 4b3a87da refactor-query-and-search -- test/datahike/test/query_stats_test.cljc
```